storey/drivers.py

```diff
@@ -179,6 +179,11 @@ def _build_feature_store_update_expression(self, aggregation_element, additional
         condition_expression = None
         expressions = []
         pending_updates = {}
+        saved_engine_words = ['min', 'max', 'sqrt', 'avg', 'stddev', 'sum', 'length', 'init_array', 'set', 'remove',
+                              'regexp_replace', 'find', 'find_all_indices', 'extract_by_indices', 'sort_array', 'blob',
+                              'in', 'true', 'false', 'and', 'or', 'not', 'contains', 'ends', 'starts', 'exists', 'select',
+                              'str', 'to_int', 'if_not_exists', 'regexp_instr', 'delete', 'as', 'maxdbl', 'inf', 'nan',
+                              'isnan', 'nanaszero', 'IF_ELSE']
```

Comment:
1. better save the words in a dictionary instead of list so you get 0(1) check instead of iterating the list each time
2. if you always compare lower case to lower case then you can save 'IF_ELSE' as 'if_else'